### PR TITLE
Configure MaxGridSum

### LINKED ISSUE #210: Setting MaxGridSum
I wish there was a way to set the "MaxGridSum" value in the constant to allow for detailed layout, for example, to create a 13-column TableList.

### PR DESCRIPTION
**Pull request by fjogeleit**
**Description**
Add Property `maxGridSize` and Setter to configure MaxGridSize, defaults to 12

**Related Issue**

closes #210 

**Checklist**
> check with "x", if applied to your change

- [x] All methods associated with structs has ```func (s *struct) method() {}``` name style. 
- [x] Wrote unit tests for new/changed features. 
- [x] Updated docs/doc.go 
- [x] Updated pkg/pdf/example_test.go 
- [x] Updated README.md 
- [ ] Updated all examples inside internal/examples 
- [x] New public methods/structs/interfaces has comments upside them explaining they responsibilities 
- [ ] Executed `make dod` with none issues pointed out by `golangci-lint` and `goreportcard-cli`

### CODE REVIEW

**Comment by johnfercher** [comment]
Hello, could you finish the checklist?

Your change needs to match ` Updated pkg/pdf/example_test.go` and  `Updated all examples inside internal/examples`

**Comment by fjogeleit** [comment]
Hey sure, I needed a quick working version for an project but I will work it out.

**Comment by fjogeleit** [comment]
Hey,

While I tried to update the PDF examples with `make examples` I got the following error `Could not save PDF: stat internal/assets/fonts/arial-unicode-ms.ttf: no such file or directory`

Because this file is not available in the repo.

With the `make dod` command I get

```
        internal/examples/utfsample/main.go
                Line 75: warning: "Tradicional" is a misspelling of "Traditional" (misspell)
golangci-lint run --config=.golangci.yml ./...
panic: load embedded ruleguard rules: rules/rules.go:13: can't load fmt

goroutine 1 [running]:
github.com/go-critic/go-critic/checkers.init.9()
        github.com/go-critic/go-critic@v0.6.1/checkers/checkers.go:58 +0x4b4
```

I used the `install.sh` to install the dependencies. And it creates `*.go.grc.bk` files as mocks - is this expected and should I rename them to replace the old mock files?

**Comment by johnfercher** [comment]
The panic is because you doesn't have the customfont inside this folder, no problem with that, just submit with all other pdf examples re-generated.

**Comment by fjogeleit** [comment]
Okay, I run `make examples` and commited all pdfs that I could recreate (except the one with the custom font).

File: internal/tablelist.go [hunk 1]
```diff
@@ -37,16 +37,22 @@ type TableList interface {
 }
 
 type tableList struct {
-	pdf  MarotoGridPart
-	text Text
-	font Font
+	pdf        MarotoGridPart
+	text       Text
+	font       Font
+	maxGridSum float64
```
Comment (johnfercher) [inline internal/tablelist.go:43]: I think that this should be an int.

**Comment by johnfercher** [comment]
Sorry for the delay in your review. Could you add an example specific for this feature? as there is an example for 'barcode', I think that should be an example for these custom grid sizes.

**Review by johnfercher — CHANGES_REQUESTED** [review]

File: internal/tablelist.go (hunk 1 above)
Comment (fjogeleit) [reply internal/tablelist.go:43]: but the default as well as all related setter functions expect a float64 value?

Comment (johnfercher) [reply internal/tablelist.go:43]: You are right. Could you just add the example with this custom grid sum?

**Review by johnfercher — CHANGES_REQUESTED** [review]

**Comment by fjogeleit** [comment]
Added an example which renders 2 basic tables with a different amount of columns based on the MaxGridSum config.

example

**Pull request merged**